build.gradle.kts

```diff
@@ -91,6 +91,7 @@ kotlin {
             dependencies {
                 implementation(kotlin("test-common"))
                 implementation(kotlin("test-annotations-common"))
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-native:1.3.7")
```

Comment:
turns out that using kotlinx-coroutines-core-common does not provide a `runBlocking` method, needed for testing coroutines. There is no runBlocking for KotlinJS, but native provides an implementation for jvm and ios.

Reply:
I think we should use `runBlockingTest` from `org.jetbrains.kotlinx:kotlinx-coroutines-test` as a `testImplementation` dependency. 

[This](https://github.com/michaelbull/kotlin-coroutines-jdbc) project has an example of using the coroutines testing library

Reply:
that is only for jvm though